common/src/main/java/com/wynntils/models/containers/ContainerModel.java

```diff
@@ -19,19 +19,27 @@
 import net.minecraft.world.item.ItemStack;
 
 public final class ContainerModel extends Model {
+    // Test suite: https://regexr.com/7jh15
     public static final Pattern ABILITY_TREE_PATTERN =
             Pattern.compile("(?:Warrior|Shaman|Mage|Assassin|Archer) Abilities");
 
     // Test suite: https://regexr.com/7b4lf
     private static final Pattern GUILD_BANK_PATTERN =
             Pattern.compile("[a-zA-Z ]+: Bank \\((?:Everyone|High Ranked)\\)");
 
+    // Test suite: https://regexr.com/7jgvf
+    private static final Pattern GUILD_MEMBER_LIST_PATTERN = Pattern.compile("[a-zA-Z ]+: Members");
```

Comment:
didnt we have this pattern already for adding the search bar?

Reply:
It's slightly different but also works

Reply:
Perhaps it's best not to duplicate the patterns then? I am not sure where it is put though..

Reply:
It'll either have to be `return SearchableContainerType.getContainerType(StyledText.fromComponent(screen.getTitle())) == SearchableContainerType.MEMBER_LIST;` or adding a getter for the title pattern in SearchableContainerType